podpac/core/coordinates/coordinates.py

```diff
@@ -442,6 +451,38 @@ def from_url(cls, url):
 
         return cls.from_definition(coords)
 
+    @classmethod
+    def from_geotransform(cls, geotransform, shape, crs=None):
+        """ Creates Coordinates from GDAL Geotransform. 
+        
+        """
+        try:
+            rcoords = RotatedCoordinates.from_geotransform(geotransform)
+            if rcoords.theta == 0:
+                raise Exception  # More appropriate to use uniform coordinates
+            coords = Coordinates([rcoords], dims=["lat,lon"], crs=crs)
+            return coords
+        except:

```

Comment:
Can you make this more specific? This is a big red flag for me because obscured exceptions are really hard to figure out.

If this is meant to catch the `rcoords.theta == 0`, then can we just do that check? If you need to catch other exceptions, too, I'd rather see something like this:

```
try:
    rcoords = RotatedCoordinates.from_geotransform(geotransform)
except <SpecificException>:
    _logger.debug(<some description of the expected/handled error>)
    return self._from_geotransform_nonrotated(geotransform)

if rcoords.theta == 0:
    _logger.debug("Rasterio source dataset does not have Rotated Coordinates")
    return self._from_geotransform_nonrotated(geotransform)

return Coordinates([rcoords], dims=['lat,lon'], crs=crs)
```

Reply:
I didn't quite take your suggestion but yes, I was being too cute (read obscure) with my exception raising.